contents/euclidean_algorithm/code/kotlin/EuclideanMod.kt

```diff
@@ -0,0 +1,18 @@
+import kotlin.math.absoluteValue
+
+fun euclidMod(a: Int, b: Int): Int {
+    var a = a.absoluteValue
+    var b = b.absoluteValue
+
+    while (b != 0) {
+        val tmp = b
+        b = a % b
+        a = tmp
+    }
+
+    return a
+}
+
+fun main() {
```

Comment:
In Kotlin, `main` should always take an array of arguments:
```suggestion
fun main(args: Array<String>) {
```